### PR TITLE
feat: database backups

### PR DESCRIPTION
**Pull request by LarsTheGlidingSquirrel**

### CODE REVIEW

File: scripts/setup_uberspace.sh [hunk 1]
```diff
@@ -77,3 +77,7 @@ if ! $(uberspace web backend list | grep -q 'http:3000 => OK, listening'); then
   exit 2
 fi
 echo 'Backend app opened to the internet'
+
+# Set cronjob for database backups
+# IMPORTANT: This overwrites existing entries!
+crontab ~/serlo-editor-as-lti-tool/uberspace/backup_cron
```
Comment (hejtful) [inline scripts/setup_uberspace.sh]: Maybe add a newline to the end of this file, to avoid any risks 😁

File: uberspace/backup_cron [hunk 2]
```diff
@@ -0,0 +1 @@
+0 2 * * * /bin/bash ~/serlo-editor-as-lti-tool/uberspace/database_backup.sh
```
Comment (hejtful) [inline uberspace/backup_cron]: Would you consider adding a comment here, which explains how often this runs?

Alternatively, a website like https://crontab.guru/ is helpful as well.

File: uberspace/database_backup.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+mysqldump edtrstag > ~/backup.sql
```
Comment (hugotiburtino) [inline uberspace/database_backup.sh]:
```suggestion
mysqldump ${USER} > ~/backup.sql
```

File: scripts/setup_uberspace.sh (hunk 1 above)
Comment (hugotiburtino) [inline scripts/setup_uberspace.sh]: How could we make a check in which environment we are? We want it only in production

Comment (LarsTheGlidingSquirrel) [reply scripts/setup_uberspace.sh]: Definitely! 😁

File: uberspace/backup_cron (hunk 2 above)
Comment (LarsTheGlidingSquirrel) [reply uberspace/backup_cron]: We can add a comment once this will probably not change any more. For now, I linked crontab.guru

**Review by hejtful — APPROVED** [review]

**Comment by hugotiburtino** [comment]
Now, as said in the [documentation](https://manual.uberspace.de/daemons-cron/), people with access to the dev email are receiving emails due to the cronjobs.

image

So let's add the MAILTO='' to the cronjob in order to only receive it in case of error

**Comment by LarsTheGlidingSquirrel** [comment]
I disabled all crontab emails for now. We have to remember to reactivate them once production is deployed.

**Pull request merged**